priv/templates/boot.eex

```diff
@@ -44,7 +72,11 @@ REL_LIB_DIR="$RELEASE_ROOT_DIR/lib"
 ERL_OPTS="<%= release.profile.erl_opts %>"
 # When stdout is piped to a file, this is the directory those files will
 # be stored in. defaults to /log in the release root directory
-RUNNER_LOG_DIR="${RUNNER_LOG_DIR:-$RELEASE_ROOT_DIR/log}"
+
+# RELEASE_MUTABLE_DIR
+RELEASE_MUTABLE_DIR=${RELEASE_MUTABLE_DIR:-$RELEASE_ROOT_DIR/running-environment}
```

Comment:
What about `var`? It vibes with your standard UNIX-y directory naming conventions, and `var/log` seems natural to me. Thoughts?